<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1032</width>
    <height>603</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>梁受力计算器</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/bridge1.ico</normaloff>:/bridge1.ico</iconset>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QLineEdit" name="forceText">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>330</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="posText">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>290</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>隶书</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>力/N</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>隶书</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>位置/M</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>470</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>440</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>集中载荷</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>集中力偶</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>均布载荷</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>线性载荷</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>10</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>长度/M</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lengthText">
   <property name="geometry">
    <rect>
     <x>99</x>
     <y>10</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>30</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ElaText">
   <property name="geometry">
    <rect>
     <x>99</x>
     <y>50</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>200</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>50</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>弹性模量/GPa</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>90</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Iz/(M^4)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="IzText">
   <property name="geometry">
    <rect>
     <x>99</x>
     <y>90</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0.05</string>
   </property>
  </widget>
  <widget class="QLabel" name="posLabel_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>隶书</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>位置2/M</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="posText_2">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>370</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="forceLabel_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>隶书</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>力2/N</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="forceText_2">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>410</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>470</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>载荷清零</string>
   </property>
  </widget>
  <widget class="QLabel" name="rResLabel">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>160</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>约束位置/m</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="rResText">
   <property name="geometry">
    <rect>
     <x>99</x>
     <y>160</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="scaleLabel">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>230</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Y轴变形相对于X坐标放大倍数:5000</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="scaleOpt">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>250</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>滚轮缩放</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="savePicButton">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>252</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>保存为图片</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveFileButton">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>280</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>导出为文件</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>20</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>载荷及挠度</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>220</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>剪力图</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>420</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>弯矩图</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>510</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string> 方向说明:载荷向下为正,力偶顺 时针为正</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="MLabel">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>166</y>
     <width>221</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>弯矩:</string>
   </property>
  </widget>
  <widget class="QLabel" name="vLabel">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>73</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>挠度:</string>
   </property>
  </widget>
  <widget class="QLabel" name="XPosLabel">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>27</y>
     <width>221</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>X坐标:</string>
   </property>
  </widget>
  <widget class="QLabel" name="FsyLabel">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>120</y>
     <width>221</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>剪力:</string>
   </property>
  </widget>
  <widget class="QLabel" name="maxFsyPos">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>461</y>
     <width>221</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>位置:</string>
   </property>
  </widget>
  <widget class="QLabel" name="MzMaxLabel">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>339</y>
     <width>221</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>绝对值最大弯矩:</string>
   </property>
  </widget>
  <widget class="QLabel" name="maxBendingPos">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>543</y>
     <width>221</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>位置:</string>
   </property>
  </widget>
  <widget class="QLabel" name="FsyMaxLabel">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>421</y>
     <width>221</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>绝对值最大剪力:</string>
   </property>
  </widget>
  <widget class="QLabel" name="bendingMaxLabel">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>502</y>
     <width>221</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>绝对值最大挠度:</string>
   </property>
  </widget>
  <widget class="QLabel" name="maxMzPos">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>380</y>
     <width>221</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>位置:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addResButton">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>190</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>增加约束</string>
   </property>
  </widget>
  <widget class="QPushButton" name="calcButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>560</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>计算并更新图形</string>
   </property>
  </widget>
  <widget class="QPushButton" name="resClrButton">
   <property name="geometry">
    <rect>
     <x>99</x>
     <y>190</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>约束清零</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>210</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>提示：在图像中点击并移动鼠标可以获取以上信息</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="sectionButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>打开截面几何计算器</string>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/1074273.png</normaloff>:/1074273.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>28</width>
     <height>28</height>
    </size>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>posText</tabstop>
  <tabstop>forceText</tabstop>
  <tabstop>posText_2</tabstop>
  <tabstop>forceText_2</tabstop>
  <tabstop>comboBox</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>pushButton_3</tabstop>
  <tabstop>lengthText</tabstop>
  <tabstop>ElaText</tabstop>
  <tabstop>IzText</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
